src/code.cloudfoundry.org/vxlan-policy-agent/converger/converger_test.go

```diff
@@ -603,48 +603,118 @@ var _ = Describe("Single Poll Cycle", func() {
 				// to validate that we get multiple errors returned, and also that we desire both
 				// the old ASG chains from failed enforces, and new ASG chains from successful enforces
 				// when cleaning up orphaned chains
-				i := 0
-				fakeEnforcer.EnforceRulesAndChainStub = func(e enforcer.RulesWithChain) (string, error) {
-					var err error
-					if i > 0 {
-						err = fmt.Errorf("eggplant")
-					}
-					i++
-					return fmt.Sprintf("%s-with-new-suffix", e.Chain.Prefix), err
-				}
```

Comment:
this looks like it was moved below, but the comment is now abandoned. should that get moved or cleaned up as well?

Reply:
cool, thanks for the notice